adam-cli/src/test/scala/org/bdgenomics/adam/cli/ADAM2FastaSuite.scala

```diff
@@ -17,18 +17,16 @@
  */
 package org.bdgenomics.adam.cli
 
-import com.google.common.io.Files
 import java.io.File
-import org.apache.spark.rdd.RDD
-import org.bdgenomics.adam.rdd.ADAMContext._
+
```

Comment:
mm my IntelliJ's ⌘⇧O defaults to a blank line between `java` imports, others, and `scala` imports:

![](https://cl.ly/3a1D2e2j1D1x/Screen%20Shot%202016-11-15%20at%201.25.58%20PM.png)

I feel like that's a style I've seen around out in the world. I guess ADAM is not set up that way, but it also seems like people are not using IntelliJ's (or anything else's) import-optimization helpers in the codebase, as I've seen multiple unused imports laying around.

Also I'm not sure if I can change that IntelliJ setting per-project :( but lmk if given all this you still want to just have "one import blob" be the rule.